content/docs/user-guide/visualization/animation/animation-editor/blending-posesubtract.md

```diff
@@ -32,6 +32,10 @@ With the **Pose Subtract** node, you can generate an additive pose. You can then
 
 ## Pose Subtract Node Attributes 
 
+<<<<<<< HEAD
 For attribute settings that are shared among the blend node types, see [Blend Node Attributes](/docs/userguide/animation/editor/blending-poses#animation-editor-blending-attributes).
 
 ![Pose Subtract node attributes.](/images/user-guide/actor-animation/animation-editor-blending-posesubtract-attributes.png)
+=======
+For attribute settings that are shared among the blend node types, see [Blend Node Attributes](blending-poses/#blend-node-attributes).
+>>>>>>> 572c5d1f0 (Fix all remaining broken links.)
```

Comment:
Not sure if these are from a current merge conflict or not, flagging them just in case.

Reply:
Weird, `git diff --check` didn't flag these. Behavior must have changed, fixed now.